### PR TITLE
[EDA] Error Settings: Store Error Setting

### PR DESCRIPTION
**Pull request by m-kinkade**
# Critical Changes

# Changes

# Issues Closed

# New Metadata
### Apex Classes
ErrorSettingsController
ErrorSettingsController_TEST
ErrorSettingsMapper
ErrorSettingsMapper_TEST
ErrorSettingsModel
ErrorSettingsModel_TEST
ErrorSettingsService
ErrorSettingsService_TEST
ErrorSettingsVMapper
ErrorSettingsVMapper_TEST
ErrorSettingsVModel
ErrorSettingsVModel_TEST

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/default/lwc/errorSettings/errorSettings.js [hunk 1]
```diff
@@ -43,17 +84,13 @@ export default class ErrorSettings extends LightningElement {
...
-        //     this.template.querySelectorAll("c-settings-row-dual-listbox").forEach((dualListBox) => {
-        //         dualListBox.resetValue();
-        //     });
-        //     this.template.querySelectorAll("c-settings-row-input").forEach((input) => {
-        //         input.resetValue();
-        //     });
-        // });
+        Promise.all([refreshApex(this.errorSettingsWireResult)]).then(() => {
+            this.template.querySelectorAll("c-settings-row-dual-listbox").forEach((dualListBox) => {
+                dualListBox.resetValue();
```
Comment (skosgi1) [inline force-app/main/default/lwc/errorSettings/errorSettings.js:89]: is dual list box needed here?

**Review by skosgi1 — CHANGES_REQUESTED** [review]
Couple of minor changes and some clean up suggestions. Everything else looks good.

File: force-app/main/default/lwc/errorSettings/errorSettings.js (hunk 1 above)
Comment (m-kinkade) [reply force-app/main/default/lwc/errorSettings/errorSettings.js:89]: Good catch!  Removing now.

**Review by skosgi1 — APPROVED** [review]
Looks good. Approved.

**Pull request merged**